graphql/resolvers.go

```diff
@@ -1502,37 +1502,31 @@ func (r *queryResolver) Projects(ctx context.Context) ([]*GroupedProjects, error
 		return nil, ResourceNotFound.Send(ctx, err.Error())
 	}
 
-	groupsMap := make(map[string][]*restModel.APIProjectRef)
-
-	for _, p := range allProjs {
-		groupName := strings.Join([]string{p.Owner, p.Repo}, "/")
-		apiProjectRef := restModel.APIProjectRef{}
-		if err = apiProjectRef.BuildFromService(p); err != nil {
-			return nil, InternalServerError.Send(ctx, fmt.Sprintf("error building APIProjectRef from service: %s", err.Error()))
-		}
-
-		if projs, ok := groupsMap[groupName]; ok {
-			groupsMap[groupName] = append(projs, &apiProjectRef)
-		} else {
-			groupsMap[groupName] = []*restModel.APIProjectRef{&apiProjectRef}
-		}
+	groupedProjects, err := GroupProjects(allProjs, false)
+	if err != nil {
+		return nil, InternalServerError.Send(ctx, fmt.Sprintf("error grouping project: %s", err.Error()))
 	}
+	return groupedProjects, nil
+}
 
-	groupsArr := []*GroupedProjects{}
+func (r *queryResolver) ViewableProjects(ctx context.Context) ([]*GroupedProjects, error) {
```

Comment:
What will this resolver be used for? Should it not also show projects that don't default? (grouped by the empty string or something)

Reply:
It does, it's be grouped under "".

Reply:
Oh I think onlyDefaultedToRepo confused me; that implies to me that we're only returning projects that use the repo default. Can you add some comments to that function?

Reply:
I'm not sure what you mean by repo default. I set it up this way based on your comment here: [5117](https://github.com/evergreen-ci/evergreen/pull/5117#discussion_r733743503). What kind of comment would be helpful here?